PIM/Integration/02_ManageAttributeSets.md

```diff
@@ -0,0 +1,272 @@
+[!!User Interface Attribute sets](../UserInterface/03b_AttributeSets.md)
+[!!Manage the attributes](./01_ManageAttributes.md)
+[!!Manage the products](../Operation/01_ManageProducts.md)
+
+
+# Manage an attribute set
+
+An attribute set consists of a group of attributes to classify certain products according to demand or production contexts. By default, the *PIM Basic Set* attribute set is predefined.
```

Comment:
maybe change group of attributes to list of attributes in order to avoid possible confusion with attribute groups?